samples/legacy_support/All_ID_Mapping_Latest.csv

```diff
@@ -2391,7 +2391,7 @@ id,Name,LegacyKey,MainIcon,Modifier1,Modifier2,ExtraIcon,FullFrame,GeometryType,
 2389,Ice Systems : Dynamic Processes : Convergence,W-S-IDC---,46110301,,,,,Point,,,
 2390,Ice Systems : Dynamic Processes : Divergence,W-S-IDD---,46110302,,,,,Point,,,
 2391,Ice Systems : Dynamic Processes : Shearing or Shear Zone,W-S-IDS---,46110303,,,,,Point,,,
-2392,Ice Systems : Dynamic Processes : Ice Drift (Direction),W---IDID--,46110304,,,,,Point,,,
+2392,Ice Systems : Dynamic Processes : Ice Drift (Direction),W---IDID--,46110304,,,,,Line,,,
```

Comment:
@csmoore : Ice Drift is a Point in 2525D.  So JMSML won't output 46110304 as a Line.  My recommendation is to add another "C" "Original" mapping for the Line version in 2525C so the "Current" mapping to 46110304 remains true to 2525D.